packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -713,6 +731,45 @@ Map<String, String> parseAdbDeviceProperties(String str) {
   return properties;
 }
 
+
+const String _kJavaHeapKey = 'Java Heap';
```

Comment:
The string 'Java Heap' isn't in the example below or in the linked docs.

Reply:
Updated to full example